OWNERS_ALIASES

```diff
@@ -14,15 +13,14 @@ aliases:
   - ankitathomas
   - bentito
   - dtfranz
-  - everettraven
   - grokspawn
   - joelanford
   - kevinrizza
   - oceanc80
   - perdasilva
   - rashmigottipati
-  - stevekuznetsov
-  - theishshah
   - tmshort
-  - varshaprasad96
   - LalatenduMohanty
+  - camilamacedo86
```

Comment:
nit: alphabetize whole list?

Reply:
sigh

Reply:
Next you'll be telling me to get rid of blank lines at the end of the file! :D

Reply:
I mean ... turnabout is fair play. 😁